resources/locales/en/translation.json

```diff
@@ -8,12 +8,18 @@
   "Nur eintragen, sofern für den Ausgang der Wahl relevant&#46;": "only enter if relevant to the outcome of the election&#46;",
   "Stimmen liegen in % anstatt absoluten Zahlen vor": "votes are in % instead of absolute numbers",
   "Verfügbare Sitze": "available seats",
+  "Daten mit Fehlermarge": "Data with error margin",
```

Comment:
I think we should consistently use "Fehlerbereich" or "Fehlermarge" and not use both. In q-election-executive we just say "mit Fehlerbereich" and not "Daten mit Fehlermarge". I think this should also be consistent between the tools

Reply:
done